marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/writer/HubDataWriterFactory.java

```diff
@@ -43,13 +44,7 @@
     public HubDataWriterFactory(Map<String, String> params, StructType schema) {
         this.params = params;
         this.schema = schema;
-
-        Properties props = new Properties();
-        params.keySet().forEach(key -> props.setProperty(key, params.get(key)));
-        logger.info("Creating HubClient for host: " + props.getProperty("mlHost"));
-        HubConfigImpl hubConfig = new HubConfigImpl();
-        hubConfig.registerLowerCasedPropertyConsumers();
-        hubConfig.applyProperties(new SimplePropertySource(props));
+        HubConfigImpl hubConfig=buildHubConfig(params);
```

Comment:
Assuming you're using Intellij, be sure to do a Reformat Code on every file - I have option-command-L as a shortcut for that, I forget if it's a default in Intellij. That'll ensure that we don't get a bunch of changes to this file in another PR when someone does run Reformat Code.

Reply:
Formatted the code